test-api-js/todo.service.js

```diff
@@ -31,10 +31,14 @@ class todoservice{
 
     delete_todo(id){
         // Your code here
+        this.todos.todo.splice(id,1);
+        return this.todos;
     }
 
     update_todo(id, todo){
         // Your code here
+        this.todos.todo.splice(id,1,todo);
```

Comment:
It would be better if we null check id and todo

Reply:
Code looks simple and clean. Can go ahead with the merge.

Reply:
Thank you. Will work on it.